UpdatePipelineLabel/UpdatePipelineLabel/LabelUpdateHelper.cs

```diff
@@ -0,0 +1,197 @@
+﻿using Azure;
+using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using ZenHub;
+using ZenHub.Models;
+
+
+
+namespace UpdatePipelineLabel
+{
+    class LabelUpdateHelper
+    {
+        private const string WorkspaceName = "NuGet Client Team";
+        private readonly GitHubClient GitHubClient;
+        private readonly ZenHubClient ZenHubClient;
+
+        private string repoName;
+        private string githubToken;
+        private string zenhubToken;
+        private int issueNumberFrom;
+        private int issueNumberTo;
+
+        //If specify a range in the command, then only run issues having issue numbers in this range.
+        //If range is not specified, run all the issues in the repository.
+        bool processAll;
+
+        //private SummaryReport summaryReport;
+
+        public LabelUpdateHelper(Options options)
+        {
+            repoName = options.Repo;
+            githubToken = options.GitHubToken;
+            zenhubToken = options.ZenHubToken;
+
+            //only run issues which is in the range, if the totall issue number exceed the rate limiting https://developer.github.com/v3/#rate-limiting
+            issueNumberFrom = options.IssueNumFrom;
+            issueNumberTo = options.IssueNumTo;
+
+            //if issueNumberTo is not specified in the command, or the "from" is larger than "to" , then process all the issues
+            if (issueNumberTo == 0 || (issueNumberFrom > issueNumberTo))
+            {
+                processAll = true;
+            }
+
+            GitHubClient = new GitHubClient(new ProductHeaderValue("nuget-update-pipeline-label"));
+            var creds = new Credentials(githubToken);
+            GitHubClient.Credentials = creds;
+            ZenHubClient = new ZenHubClient(zenhubToken);
+
+            //summaryReport = new SummaryReport();
+        }
+
+        public async Task UpdateLabel()
+        {
+            Dictionary<int, string> issueLabelMap = await GernerateIssueLabelMap();
+
+            string[] repoParts = repoName.Split('/');
+
+            List<string> pipelineLabels = new List<string>();
+            pipelineLabels.Add(IssueLabels.NewIssues);
+            pipelineLabels.Add(IssueLabels.Icebox);
+            pipelineLabels.Add(IssueLabels.Backlog);
+            pipelineLabels.Add(IssueLabels.InProgress);
+            pipelineLabels.Add(IssueLabels.InReview);
+            pipelineLabels.Add(IssueLabels.Validating);
```

Comment:
Can't you build Lists like this?

```suggestion
            List<string> pipelineLabels = new List<string>()
            {
                IssueLabels.NewIssues,
                IssueLabels.Icebox,
                ...etc
            };
```

Reply:
Thanks! Changed.